app/Repositories/PostRepository.php

```diff
@@ -93,6 +93,11 @@ public function create(array $data, $signupId, $authenticatedUserRole = null)
             'details' => isset($data['details']) ? $data['details'] : null,
         ]);
 
+        // If there is a created_at property, fill this in (e.g. if created_at is sent when creating a record with the importer app).
+        if ($data['created_at']){
```

Comment:
Can we limit this functionality to admins? (Like how we do with `status` and `source` below?

Reply:
oh yes great call! Updated :)